include/dts-functions.sh

```diff
@@ -1845,3 +1813,29 @@ ask_for_confirmation() {
     esac
   done
 }
+
+parse_config() {
+  local vendor="$1"
+  local system_model="$2"
+  local board_model="$3"
+  json_file="$BOARD_CONFIG_PATH/configs/$(echo "$vendor" | tr '[:upper:]' '[:lower:]' | sed 's/ /_/g').json"
```

Comment:
It does not have to block this PR, but my impression is: if we already have python anyway and use it in dts, I would personally write a simple python3 script and execute it there.

I wonder if one can easily understand this jq code without a headache.